ffmpegdecklink.rb

```diff
@@ -1,9 +1,8 @@
 class Ffmpegdecklink < Formula
   desc "FFmpeg with --enable-decklink"
   homepage "https://ffmpeg.org/"
-  url "https://ffmpeg.org/releases/ffmpeg-3.4.1.tar.bz2"
-  sha256 "f3443e20154a590ab8a9eef7bc951e8731425efc75b44ff4bee31d8a7a574a2c"
-  revision 3
+  url "https://ffmpeg.org/releases/ffmpeg-3.4.2.tar.bz2"
```

Comment:
does 3.4.2 include the SIGPIPE patch used below. If so the patch should be removed in the same commit.

Reply:
I have the _impression_ that this has not been back-ported (3.5 should be released soon), but I will try to check. If done, then I’ll clear the patch here.

Reply:
No, I just have checked the code:
```
signal(SIGPIPE, sigterm_handler); /* Termination (pipe closed).  */
````
is not included in 3.4.2.